.npmrc

```diff
@@ -0,0 +1 @@
+puppeteer_skip_chromium_download=true
```

Comment:
I think you can just add the flag `--production` here: https://github.com/asyncapi/playground/blob/master/Dockerfile#L12.

So it would be `npm install --production` and it will skip devDependencies. Can be either this or set NODE_ENV=production on the Dockerfile too.

More info: https://docs.npmjs.com/cli/v6/commands/npm-install

Reply:
I just realized it's not a direct dependency. Still, give it a try because it may avoid installing devDependencies of the dependencies 😅

Reply:
but puppeteer is not a devDependency of html-template, it is used to generate PDF from HTML, so it is direct dependency of a dependency

Reply:
👍